torch/csrc/utils/pybind.h

```diff
@@ -190,7 +190,7 @@ struct type_caster<c10::DispatchKey>
 template <>
 struct type_caster<c10::SymInt> {
  public:
-  PYBIND11_TYPE_CASTER(c10::SymInt, _("SymInt"));
+  PYBIND11_TYPE_CASTER(c10::SymInt, _("torch._prims_common.IntLike"));
```

Comment:
How come not torch.SymInt? Though I guess technically this is inaccurate

Reply:
I wasn't too sure about this one. And I just realized `IntLike` isn't a `Union` so this wouldn't work anyway. Looking at https://github.com/pytorch/pytorch/blob/master/torch/csrc/utils/pybind.cpp#L23 it looked like it could return a plain `int64_t`.

Lmk what you think is more correct.

Reply:
this sg